.github/workflows/deploy-pages.yml

```diff
@@ -30,6 +30,11 @@ jobs:
           sudo apt-get install -y texlive texlive-fonts-extra latexmk
       - name: Fetch Yarn dependencies
         run: yarn install
+      - name: Clone translated_xmls
+        run: |
+          git clone -b translated_xmls https://github.com/source-academy/sicp.git translated_xmls
+          mv translated_xmls/* xml/
+          rm -r translated_xmls
```

Comment:
Why are we cloning from a specific branch instead of merging it to the default branch?